Childrens-Social-Care-CPD/Views/Shared/_VideoResource.cshtml

```diff
@@ -8,9 +8,34 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters">
-        <video id="videoPlayer" preload="none">
-            <source src="@Model.Video.File.Url">
-        </video>
+        @if(!string.IsNullOrEmpty(Model.EmbededTitle) && !string.IsNullOrEmpty(Model.EmbededSourceUrl))
+        {
+            <h2>
```

Comment:
Seems weird to have a title only for embedded videos? Also I can't see a h2 title in any of the designs (I might have missed one), but if there is somewhere then we should check the design consistency across the pages.

Reply:
Fixed